website/src/components/ViewEventDrawer/ViewEventDrawer.jsx

```diff
@@ -0,0 +1,328 @@
+        message: newMessage,
+      };
+      fetch('https://us-central1-rails-students.cloudfunctions.net/addmessage', {
+        method: 'POST',
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(reqData),
+      })
+        .then(res => res.json())
+        .then((result) => {
+          message.info(result.message);
+          if (result.message === 'Added comment.') {
+            const getEventReqData = { eventUid, classUid, uid };
+            fetch(
+              'https://us-central1-rails-students.cloudfunctions.net/geteventdetails',
+              {
+                method: 'POST',
+                headers: {
+                  Accept: 'application/json',
+                  'Content-Type': 'application/json',
+                },
+                body: JSON.stringify(getEventReqData),
+              },
+            )
+              .then(res => res.json())
+              .then((getEventResult) => {
+                if (getEventResult.message === 'Found event') {
+                  const nxted = getEventResult.eventData;
+                  nxted.university = university;
+                  nxted.classUid = classUid;
+                  nxted.eventUid = eventUid;
+                  nxted.discussions.forEach((ed, i) => {
+                    ed.senderName = ed.fromName;
+                    ed.id = (ed.fromName === displayName) ? 0 : ed.fromName;
+                    nxted.discussions[i] = ed;
+                  });
+                  this.setState({
+                    eventData: nxted,
+                    loading: false,
+                    newMessage: '',
+                  });
+                }
+              })
+              .catch((err) => {
+                console.log(err);
+                message.error(err.message);
+              });
+          } else {
+            this.setState({ loading: false });
+          }
+          console.log(result);
+        })
+        .catch((err) => {
+          console.log(err);
+          message.error(err.message);
+        });
+    } else {
+      this.setState({ loading: false });
+      message.info('Message can\'t be empty.');
+    }
+  }
+
+  render() {
+    const { visible, onClose } = this.props;
+    const {
+      eventData, file, loading, newMessage, enterToSend, displayName,
+    } = this.state;
+    return (
+      <Drawer
+        title={eventData ? eventData.title : 'Loading event...'}
+        placement="right"
+        width="500"
+        closable
+        onClose={onClose}
+        visible={visible}
+      >
+        {
+          eventData
+            ? (
+              <div style={{
+                width: '100%', height: '90%', display: 'flex', flexDirection: 'column',
+              }}
+              >
+                <h4>Due Date</h4>
+                <p>{`${new Date(eventData.dueDate).toDateString()} at ${new Date(eventData.dueDate).toLocaleTimeString()}`}</p>
```

Comment:
Can you use moment.js for this we already use it in the calendar and eventview components to  handle the display  of time and date strings. Just for consistency's sake.

Reply:
moment is just a wrapper for date, it would just call on the same function.